pipelines.go

```diff
@@ -73,11 +73,11 @@ func (c *Client) UpsertPipeline(p Pipeline, id string) error {
 	var unused interface{}
 	if id == "" {
 		if err := c.PostWithRetry(c.pipelinesURL(), ApplicationJson, p, &unused); err != nil {
-			return fmt.Errorf("could not create pipeline '%s' in app '%s' - %v", p.Name, p.Application, err)
+			return fmt.Errorf("could not create pipeline '%s' in app '%s': %w", p.Name, p.Application, err)
```

Comment:
This uses the new [`%w` verb](https://blog.golang.org/go1.13-errors) that allows `errors.As()` to work in dinghy